### PR TITLE
add: contribution doc and fix use_latest template tag

### PR DESCRIPTION
**Pull request by deepakdinesh1123**

### CODE REVIEW

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10196390026). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10196402863). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/apps/shared/tenancy/templatetags/zango_filters.py
Comment (kc-diabeat) [inline backend/src/zango/apps/shared/tenancy/templatetags/zango_filters.py:1]: Can we also add the docstring for this function, including the rationale and the usage pattern? @shahharsh176

Comment (kc-diabeat) [reply backend/src/zango/apps/shared/tenancy/templatetags/zango_filters.py:1]: Also not sure why we are including this change in this PR..We will  be having another PR where the template file for app panel is being updated to use this template tag. Shouldn't we have this change in that PR?

**Review by kc-diabeat — CHANGES_REQUESTED** [review]

File: backend/src/zango/apps/shared/tenancy/templatetags/zango_filters.py
Comment (shahharsh176) [reply backend/src/zango/apps/shared/tenancy/templatetags/zango_filters.py:1]: It's already merged, that's why I included the fix in this PR

Comment (shahharsh176) [reply backend/src/zango/apps/shared/tenancy/templatetags/zango_filters.py:1]: @kc-diabeat added the docstring

**Review by kc-diabeat — APPROVED** [review]

**Pull request merged**